projects/gnomad/deploy/gnomad-deployment-prod.yaml

```diff
@@ -22,7 +22,7 @@ spec:
           fsType: ext4
           readOnly: true
       containers:
-      - name: gnomad-p-serve
+      - name: gnomad-d-serve
```

Comment:
This line doesn't seem like it should change, since this file contains production configuration.